src/metabase/db/data_migrations.clj

```diff
@@ -83,18 +82,6 @@
         :group_id (:id (perm-group/admin))
         :object   "/"))))
 
-;; add existing databases to default permissions groups. default and metabot groups have entries for each individual
-;; DB
-(defmigration ^{:author "camsaul", :added "0.20.0"} add-databases-to-magic-permissions-groups
-  (let [db-ids (db/select-ids Database)]
-    (doseq [{group-id :id} [(perm-group/all-users)
-                            (perm-group/metabot)]
```

Comment:
The original version also created permissions entries for the metabot group. Not doing this here since the MetaBot group is being removed in #20565